### DISCUSSION TITLE
iOS autocomplete on input x-model leaves input blank

### DISCUSSION THREAD

**Discussion by blakeaross** [alpinejs/alpine#1968] [7. Bugs (archived)]
Autocomplete of an input on iOS does not trigger an update of x-model. View the pen below for demonstration and description of the bug.

https://codepen.io/blakeaross/pen/zYwVpZG

Here's a real-life story for why this matters: I'm the lead front-end developer for my agency, and we are creating a website for a large national brand. We are using Alpine on this project. I am creating custom form elements for the site, including floating label form elements. The convention for floating label inputs is that when a user focuses on an input, the label shifts to the top, leaving the user able to see what is being written. Alpine is working wonderfully for that interaction, but unfortunately, autocomplete not filling the input is leading to an unacceptable UX issue. Any thoughts for how to resolve this bug?

**Comment by calebrunion** [comment]
This is yet another mention of the same bug. @calebporzio, this has still not been addressed.

Other discussions that mention this bug:
1. #1909
2. #1862
3. #1810 
4. #1501

**Comment by MuzafferDede** [comment]
- #1969 should fix this
